dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/BundleBuilder.java

```diff
@@ -67,4 +76,20 @@ public Bundle build() throws SQLException, AuthorizeException {
 
         return bundle;
     }
+
+    public static void deleteBundel(UUID uuid) throws SQLException, IOException {
```

Comment:
This method is misspelled.  It should be `deleteBundle`.  Obviously, you'll need to also cleanup the places it's used in ITs.